packages/dev/core/src/Engines/WebGPU/webgpuShaderProcessorsGLSL.ts

```diff
@@ -68,9 +68,9 @@ export class WebGPUShaderProcessorGLSL extends WebGPUShaderProcessor {
     }
 
     public varyingCheck(varying: string, isFragment: boolean) {
-        const outRegex = /(flat\s)?\s*out /;
-        const inRegex = /(flat\s)?\s*in /;
-        const varyingRegex = /(flat\s)?\s*varying /;
+        const outRegex = /(flat\s)?\s*\bout\b /;
+        const inRegex = /(flat\s)?\s*\bin\b /;
+        const varyingRegex = /(flat\s)?\s*\bvarying\b /;

```

Comment:
You should remove the trailing spaces, as they are not needed anymore.

Reply:
@Popov72  Thanks, I've removed the trailing space